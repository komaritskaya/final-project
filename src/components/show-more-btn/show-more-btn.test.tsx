import React from 'react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import ShowMoreBtn from './show-more-btn';

const mockStore = configureStore([]);

it('Should ShowMoreBtn component render correctly', () => {
  const store = mockStore();

  const node = renderer.create(
    <Provider store={store}>
      <ShowMoreBtn />,
    </Provider>,
  ).toJSON();

  expect(node).toMatchSnapshot();
});
