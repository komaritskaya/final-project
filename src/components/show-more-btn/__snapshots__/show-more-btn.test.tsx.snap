// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ShowMoreBtn component render correctly 1`] = `
Array [
  <div>
    <button
      onClick={[Function]}
      type="button"
    >
      Show more
    </button>
  </div>,
  ",",
]
`;
